### DISCUSSION TITLE
Creating a priority encoder

### DISCUSSION THREAD

**Discussion by benkyousec** [hneemann/Digital#1154] [Q&A] [answered]
I need help with understanding how to wire the *num* and *any* outputs for a priority encoder. 
I am trying to replicate an 8 to 3 priority encoder like this.
![image](https://user-images.githubusercontent.com/24224821/236255390-4e4c2299-35ad-45aa-a07c-a2ffe7ca6e29.png)

**Comment by hneemann** [comment, marked as answer]
Like this?
![z](https://user-images.githubusercontent.com/7190563/236259836-1da4165b-465c-4e97-98ff-8666d181d108.png)

**Reply by benkyousec** [reply]
Thanks!
